packages/config/src/layer2s/fraxtal.ts

```diff
@@ -1,17 +1,62 @@
 import { EthereumAddress, UnixTime } from '@l2beat/shared-pure'
 
-import { underReview } from './templates/underReview'
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { opStack } from './templates/opStack'
 import { Layer2 } from './types'
 
-export const fraxtal: Layer2 = underReview({
-  id: 'fraxtal',
+const discovery = new ProjectDiscovery('fraxtal')
+
+const upgradeability = {
+  upgradableBy: ['ProxyAdmin'],
+  upgradeDelay: 'No delay',
+}
+
+export const fraxtal: Layer2 = opStack({
+  daProvider: {
+    name: 'External',
+    riskView: {
+      value: 'External - Fraxtal Custom Module',
```

Comment:
```suggestion
      value: 'External',
```

Reply:
are there any DA attestations published? if yes it should be specified, if no same thing, see Celestia for example
![image](https://github.com/l2beat/l2beat/assets/30298476/87624032-d4d0-4f7a-8e63-13b1bedd74a4)

Reply:
Thanks, added